### ISSUE TITLE
Some test(s) taking over 10 or 20 minutes to complete

### ISSUE THREAD

**Issue by jdenoc** [jdenoc/money-tracker#40]
It seems that there is a test that can take over 10 or even 20 minutes to complete.

Here is some example _output of the 20 minute timeout failure_:
```
$ travis_wait vendor/bin/phpunit --coverage-text

Timeout (20 minutes) reached. Terminating "vendor/bin/phpunit --coverage-text"
The command vendor/bin/phpunit --coverage-text exited with 137.

Log:
PHPUnit 5.7.23 by Sebastian Bergmann and contributors.

...............................................................  63 / 127 ( 49%)
............................................

The command "travis_wait vendor/bin/phpunit --coverage-text" exited with 137.
```

**We need to identify the test causing timeouts.**

**Comment by jdenoc** [comment]
While this was fixed in [0.2.0](https://github.com/jdenoc/money-tracker/releases/tag/0.2.0), we're going to wait till at least the next release until we're confident this has been fully resolved.

**Comment by jdenoc** [comment]
Looks like this is no longer an issue
